### DISCUSSION TITLE
synOCR does not yet have the necessary permissions

### DISCUSSION THREAD

**Discussion by gref1966** [geimist/synOCR#43] [Q&A] [answered]
Hallo, 
ich habe synOCR runtergeladen und manuell auf meiner Synology installiert, vorher noch Docker natürlich installiert.
Beim Öffnen von von synOCR erhalte ich aber im "Overview" Fenster den Hinweis in rot markiert:

A T T E N T I O N:
synOCR does not yet have the necessary permissions!
Please create a task in the task planner with the following script (as user root)
(the task must be executed at least once) /usr/syno/synoman/webman/3rdparty/synOCR/synOCR-start.sh

![Screenshot 2023-12-04 164556](https://github.com/geimist/synOCR/assets/79963099/da71ce11-aaaf-41b5-b535-a774a3a2c1b4)

**Comment by geimist** [comment]
Ja, hier ist eine Useraktion gefordert, da ab DSM7 Drittanbieter Pakete keine Rootberechtigung mehr haben dürfen. Das lässt sich leider nicht vermeiden. 

Der geforderte Befehl setzt den Paketuser in die Gruppe `administrators` und in die neu erstellte Gruppe `docker`.

**Reply by gref1966** [reply]
Ok, mein User mit dem ich unter DSM 7 den Container Manager installiert habe gehört zur Gruppe der Administratoren.
Aber eine Gruppe "docker" gibt es bei mir nicht.
Muss man die selber erstellen?

**Reply by geimist** [reply, marked as answer]
Frage an mich: warum habe ich hier nicht geantwortet? 🤷‍♂️
Sorry, dennoch die Antwort zur Dokumentation: Die Gruppe `docker` wird durch das einmalige Ausführen des Startskriptes über den User `root` erstellt, falls sie noch nicht besteht. Es sind also keine weiteren Schritte seitens des Users erforderlich.
